### PR TITLE
795 chase oomkill error on science self hosted ci

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: _e2e/raw2science.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
...RGdnRVBBRENDQVFvQ2dnRUJBTW5YCms4V0oraFZOZFB5NDhaWFp4UHpua2gvbGVWTFJONi9xL1gxM3JOWmRpaVY3OUJYeFNTc1dSem1CUGk0TGhEU0EKMnlSVjF2aW5xVGVzampZZWJYTldFdTV0M24vcEQ5Vk0ySmVPSkg5T0tKOWd2TU80OUVkamFLWldsRDk3eG4rTApkYU9ZQStSeENjcExkN0Rwang2NmlScm81S2Q0czh2MEthaUVPTktESFNSQ1V1Z1JYZ0J3UldLcDBNZ0tsK3R4CkdBVEY0Sk83WXoxUzVKR0pVRmozR1ViSmZrWHlIVDUxTWN2YmFnamIzTkRiRWpZNnJPVExKanE5KzY5T1VBQ0UKTUdzdFo5b0puOXJTdVNBeFN1V3BCZ3BYVkkyLzd5WVZGZW9WUkZWenAyenhGZVc0V3l6YnRGT09Ea09kMk5ENAp4c21tSGVBN0FkUHo2cStiT3pjQ0F3RUFBYU5aTUZjd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZEdmdVdU1oKzhoYkNyVkI4eDJkOFdvUjJ1cm1NQlVHQTFVZEVRUU8KTUF5Q0NtdDFZbVZ5Ym1WMFpYTXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBRDlGSXZkLzlZQytVaEltS0VJagpqK2RVY3RYYlorZjU5clRBU1l1WDFPUCtwbmVhNVNTcnR5Y0RLTUxPZ2doL3BHY0xpQW5KMnJYWmlTTG40eEVkCmtWL2xKaE92T0JvK0VqbmhzUkZLRkI5MW5La0toVWt0ejFUL0FJeFBCSm5RT3NEemF1UTN1NTk2bW94eHh5OUkKeTRWT0FpbVltbHRRclVtR3Y5bVlEaEJaMUlPRkZuS3gxK0daNW9YRllqQVo3aTRZK3hNZkRYT21td0pjTEVBTAo3Z0duUjBkOFNGVUFaSitqU3NzMVoyMFdFbXNyVC9ndFJ1UDFWQXpDajlXMEtJZ1BBZnUzQnJTdHhUY2FxcTlyClhpdE5mY3I4UXordDMzbjMrenBObUV6NUNPWDBMVmVlSE0yNkpDTjlZOHh2eElLVThBL2VsTS9waUlra0V4TysKNkNJPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```
Comment (JulienPeloton) [inline _e2e/raw2science.sh]: Just to understand, where this certificate comes from and how it has been generated?

File: _e2e/raw2science.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
...VzNiQW1nQ0xiQkVtSDFEaWlMYzJFWnduYkJEUFVjUgpSaUNHVjgxNXdkcTIrc3lHSDNYRnJJWUs5Mk5haVgrcVhlakVJKy9kYmNJMDJXcmRYVnJ5Z0JDbXluK3U4ZVNWClVlSDFmT2Z1TjVNUzFBcEtjS1lDOVF5QmZPb2VxekJSUEJPRGpQazJXSVpMajlQNTAzYnRXK1hMQW9HQkFNbDYKR1RhNnhPTG9jRm5sRGhIVnhlWGU0SXpEK0NSbkxicWpjNmpISHB6eUZEOHVTZEczaDBUU2tDSk1UTUdENnJXZApOOWJxUG0xbXBqdkJSNmlub0NTZW1FUXJoNEhpRmxLbDBsUTJjM2EzQXFWRGFnRUxralh1dDZ2eFY2dmlYdzBKClRNWEZiQnN6REQzRS9BVm55ZmZVazN1TnR5d3FDZlNYMW9rVzhNdUxBb0dCQUlaQ2hybVF4SG01UVIyZjZpSVQKY1dvSE13TGFxc3hKSVhYcjAzSEtLWlQ2OGRXeE9XSVRMT0w4NU9Bc1BtcnlQK0xiL043OWQwZTdUcTNUeVN3cgphUEEyb1VsWnV1TDZHNWlTQUNXc01QNzh5QTZjZGhwK3ZEVnB4c0hDbnJkZU1zZXF0NVZ2ZmhHWUZURGJuRE52ClZWUzZYN1VPNmpTdHpyNUZrQnpMMmN5REFvR0FlSUt2MGl5azE1UnduMVNDdGVoSUZ0YlpPVmwrYzBmRDhuOFMKRDBJSzdaVmV4dXZJKzhmckFSOXhWeEtNVnJPeEVEQ1oxZE1HeENsQ0JUdjJ3TXFDMzJSOWIwa2FKSFkzSS9oTApkbE9iZk1hNnJsOUVhSnVqT044d3ZZZitDTEcrYU8zSEhnU1lSU3lqaGM3dGI4NUU0QmdtbnlDRkQ4V1ZidGZqCld0eDk2MEVDZ1lCWkQvY1UwUWNxL3NWUG9xbC9YcjErTVNja1UwSlA0ay9yWWNEN0RNQ3UzQi92Vm9IMU5jcVIKNUJxS1hKSXVteVRUejVvbk5YbVR5ZTFmcEFtMVFqMGU3amUrK0I1TW9hWm9WTXZZZWxGb3owS3EwcjFVRy9pZQpHSHZYdzZ3UEJXU1lITGUyQ0JSTlBmbktrQy9qTHJYQkt1eTFkQnR0dW9vNjhiRWUwdzNxSUE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```
Comment (JulienPeloton) [inline _e2e/raw2science.sh]: Same question for these certificates

**Review by JulienPeloton — APPROVED** [review]
Thanks @fjammes -- it looks good to me! I only have questions about the certificates generation (see below).

File: _e2e/raw2science.sh (hunk 1 above)
Comment (fjammes) [reply _e2e/raw2science.sh]: Good point, this certificate is useless for e2e test. I have removed it.

Comment (JulienPeloton) [reply _e2e/raw2science.sh]: thanks!

**Pull request merged**